ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -354,8 +354,7 @@ impl Blockstore {
     ) -> Result<()> {
         let mut index0 = self.transaction_status_index_cf.get(0)?.unwrap_or_default();
         let mut index1 = self.transaction_status_index_cf.get(1)?.unwrap_or_default();
-        let to_slot = to_slot.saturating_add(1);
-        for slot in from_slot..to_slot {
+        for slot in from_slot..to_slot.saturating_add(1) {
```

Comment:
How about we simply use `..=`?

    for slot in from_slot..=to_slot {

Reply:
Looked up the original PR that added this; it seems you intended to use `..=` back then as well 😁 . Changed
https://github.com/solana-labs/solana/pull/27201/files#r960136216